README.md

```diff
@@ -20,15 +20,15 @@ This action publishes your app on Fission.
 
 ## Inputs 
 
-### `machine_key`
+### `machine_key` (`fission_key`)
```

Comment:
The input MUST be `machine_key` (because the entrypoint script uses this). The GH secret name, however, can be anything - the user defines which secret to assign to the machine key in their workflow.

I had intentionally used a `FISSION_KEY` secret name in the example to illustrate this but perhaps it just creates confusion?